src/plugins/data/public/actions/select_range_action.ts

```diff
@@ -27,8 +27,8 @@ export const ACTION_SELECT_RANGE = 'ACTION_SELECT_RANGE';
 
 export function createSelectRangeAction(
```

Comment:
let's rename to createSelectRangeActionDefinition to be explicit

Reply:
Done.